File: solutions_5658282861527040_0/Python/csurfer/1B_B.py
from io import open

def process(inp):
  a, b, k = map(int, inp.readline().split(' '))
  cp = []
  for i in xrange(a):
    for j in xrange(b):
      cp.append([i, j])
  cnt = 0
  for el in cp:
    if el[0] & el[1] < k:
      cnt += 1
  return cnt

def output(strOut, out):
  print strOut
  out.write(u"{}\n".format(strOut))

def main():
  inp = open("input.txt")
  out = open("output.txt", "wb+")
  tC = int(inp.readline())
  tc = tC
  while tc > 0:
    tc -= 1
    ans = process(inp)
    output(u"Case #{}: {}".format(tC-tc, ans), out)
  inp.close()
  out.close()

if __name__ == "__main__":
  main()

File: solutions_5658282861527040_0/C++/Zip753/1B_B.cpp
#include <cstdio>

int main() {
	freopen("input.txt", "r", stdin);
	freopen("output.txt", "w", stdout);
	int qwe;
	scanf("%d", &qwe);
	for (int tt = 1; tt <= qwe; tt++) {
		int ans = 0;
		int a, b, k;
		scanf("%d%d%d", &a, &b, &k);
		for (int i = 0; i < a; i++)
			for (int j = 0; j < b; j++) {
				int x = (i & j);
				if (x < k)
					ans++;
			}
		printf("Case #%d: %d\n", tt, ans);
	}
	return 0;
}
